### DISCUSSION TITLE
Collections VS Pages

### DISCUSSION THREAD

**Discussion by ottz0** [statamic/cms#7229] [General]
I'm using the Pro version of Statamic and have setup a multi site. I'm using the multisite to separate business functions. 

- Main site
- Resources such as case studies, blogs, business articles
- Products

I'm still a bit confused about the Collections and Pages. Can anyone help me try to understand the best uses cases for each?

 I understand that pages are collections and collections and pages. At this stage my main site uses pages and my cases studies are also pages, but i'm going to put my articles as collections so I can share these easier throughout my multisite.

**Comment by duncanmcclean** [comment]
Basically, collections are your different types of content. Similar to 'post types' if you've ever used WordPress.

Each collection has its own set of fields, along with other settings (like URL structure, publish dates or not...)

Does that help at all?
